docs/intro_testers.rst

```diff
@@ -17,10 +17,10 @@ Terminology
 ===========
 
 In the abstract, each set of tests is defined by a `suite`. All of our suites
-live in the `ceph-qa-suite repository
-<https://github.com/ceph/ceph-qa-suite/>`__, in the `suites` subdirectory. Each
-subdirectory in `suites` is a suite; they may also have "sub-suites" which may
-aid in scheduling, for example, tests for a specific feature.
+live in the `ceph/qa directory
```

Comment:
they actually live in `ceph` repository in `qa/suites` directory and release based test code is in `qa/tasks` directory.

Reply:
Better now? Not sure if I should mention `qa/tasks` here given that it was not mentioned before. propsals welcome